index.html

```diff
@@ -1108,7 +1108,7 @@
         <dfn>PaymentDetailsBase</dfn> dictionary
       </h2>
       <pre class="idl">
-        dictionary PaymentDetailsBase {
+        dictionary PaymentDetailsBase : Localizable {
```

Comment:
I don't think it should go here if it only applies to PaymentDetailsUpdate. Instead you're going to have to just duplicate the two things (or use a partial dictionary PaymentDetailsUpdate).

If you put it here then PaymentDetailsInit gets it, and in particular that means passing `{ get dir() { throw new Error("boo"); }` to places that expect a PaymentDetailsInit must throw.